src/azure/queue.ts

```diff
@@ -0,0 +1,42 @@
+import * as azure from "@azure/functions";
+
+import * as t from "io-ts";
+
+import * as E from "fp-ts/Either";
+import * as O from "fp-ts/Option";
+import { pipe } from "fp-ts/function";
+
+import { parse } from "fp-ts/Json";
+
+import { validate } from "../validation";
+import { trigger, InvalidTriggerError } from "./trigger";
+
+const isQueueTriggeredFunctionContext = (
+  ctx: azure.Context
+): ctx is azure.Context & {
+  bindingData: {
+    QueueTrigger: string;
```

Comment:
why not

```suggestion
    queueTrigger: string;
```

Reply:
I didn't choose the key, it's microsoft's fault 😢

https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-storage-queue-trigger?tabs=in-process%2Cextensionv5&pivots=programming-language-javascript#message-metadata

inconsistent compared to the other keys...